decidim-elections/app/controllers/decidim/elections/trustee_zone/application_controller.rb

```diff
@@ -14,6 +14,11 @@ class ApplicationController < ::Decidim::ApplicationController
 
         before_action :ensure_configured_bulletin_board!
 
+        register_permissions(::Decidim::Elections::TrusteeZone::ApplicationController,
+                             ::Decidim::Admin::Permissions,
+                             ::Decidim::Elections::Permissions,
```

Comment:
Can we reorder this? I think it's better to have it from more specific to less specific, as it makes more sense and also it's consistent with other modules (see examples for [consultations](https://github.com/decidim/decidim/blob/9eab73f43cdd78df08b1efcbd2386b36ac5e9ca9/decidim-consultations/app/controllers/decidim/consultations/application_controller.rb#L10), and [votings](https://github.com/decidim/decidim/blob/9eab73f43cdd78df08b1efcbd2386b36ac5e9ca9/decidim-elections/app/controllers/decidim/votings/application_controller.rb#L10))

```suggestion
                             ::Decidim::Elections::Permissions,
                             ::Decidim::Admin::Permissions,
```